kubernetes/client/api_client.py

```diff
@@ -18,6 +18,7 @@
 import os
 import re
 import tempfile
+import dateutil.parser
```

Comment:
This file is auto-generated by openapi-generator. Therefore this change won't persist when we re-generate the client. 

This change has been made upstream https://github.com/OpenAPITools/openapi-generator/pull/4316 v4.2.1. The change will be picked up when we upgrade to use the newer version (currently we are at v3.3.4)